lib/compat/wordpress-6.1/blocks.php

```diff
@@ -205,3 +205,33 @@ function gutenberg_block_type_metadata_multiple_view_scripts( $metadata ) {
 	return $metadata;
 }
 add_filter( 'block_type_metadata', 'gutenberg_block_type_metadata_multiple_view_scripts' );
+
+
+
+
+/**
+ * Registers the metadata block attribute for block types.
+ *
+ * Once 6.1 is the minimum supported WordPress version for the Gutenberg
+ * plugin, this shim can be removed
+ *
+ * Doesn't need to be backported into Core.
```

Comment:
Why this code does not required core backporting?

Reply:
Ah that would be me copy/pasting I think.

My understanding would be that we _would_ need to backport this change to Core, but I don't have a tonne of experience in this area.

Perhaps @adamziel or @noisysocks can advise?

Reply:
It seems we need backporting here, I added the label, and we can remove this comment.